.github/ISSUE_TEMPLATE/LRColloqSpeakerNom.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Long Range Colloquium speaker suggestion
+about: Nominate a speaker for LR Colloquium series
+title: "Please specify the title"
+labels: nomination
```

Comment:
The label should exist. I think this is perhaps too specialized. We can create a label for the long range colloquium though.